extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -190,8 +191,9 @@ export class AmpA4A extends AMP.BaseElement {
 
   /**
    * @param {!Element} element
+   * @param {number} upgradeDelayMs
    */
-  constructor(element) {
+  constructor(element, upgradeDelayMs) {
```

Comment:
This is essentially what Dima is advocating where we create a function on BaseElement that allows for passing the value.  I'm fine with either way.  @lannka what do you think?